FOGL-2270 sinusoid poll mode

ashish-jabble opened the pull request:
**Known_issues:**

1) **foglamp_version:**-> This plugin is compatible with foglamp **develop branch** only.
2)  _git tag_ but we can't at the moment due to FOGL-2236


**NOTE:** - 
1) Async version back up here https://github.com/foglamp/foglamp-south-sinusoid/tree/async_backup
2) **pollInterval** config item is no more use at plugin level as it is handled by southAdvanced Config and "**restart**" key is removed from reconfiguration as this key is used only by Python South server.

MarkRiddoch commented:
This will be included in the next release, so it is fine that it depends on develop to be able to run. We should set the required version to 1.5.

As poll interval is no longer used then we should remove it.

praveen-garg commented:
We can not set to 1.5 today, unless we have FogLAMP's version set to 1.5; otherwise debian package will not install

ashish-jabble commented:
@MarkRiddoch Please review the [commit](https://github.com/foglamp/foglamp-south-sinusoid/pull/21/commits/bfdbbf5b17e128d9ddf246f418892ac7368d4115). I have changed plugin_version from **1.0 to 2.0.0** as we now Poll mode and also changed foglamp_version to **1.5** as this plugin is now compatible with develop ONLY(i.e due to displayName config item addition). So keeping that fact debian installation is failed at the moment as we have **1.4.1** in foglamp develop . Once foglamp 1.5  new version is available debian installation will work without any change at plugin level.

praveen-garg commented on VERSION.south.sinusoid:
```diff
@@ -1,2 +1,2 @@
-foglamp_version>=1.3
\ No newline at end of file
+foglamp_south_sinusoid_version=2.0.0
+foglamp_version>=1.5
```
kindly change to 1.4 (as it will go to develop only), we will fix it. when FORMAL release will happen.
